madhtl/Tutor2_s28795
# chore: refactor database setup to use utility functions

This PR includes:
- Utility functions for database operations (connect, create table).
- Utility functions for image transformations (convert dict to bytes).
- Refactored `database_setup.py` and `data_cleaning.py` to use reusable code from `utils`.
- Updated README with setup and utility usage instructions.

### Key Changes
1. Modularized database setup logic in `utils/db_utils.py`.
2. Added data transformation helpers in `utils/data_utils.py`.
3. Updated `database_setup.py` to use utility functions.
4. Updated `data_cleaning.py` to handle missing values more efficiently.

### Testing Instructions
1. Run `database_setup.py` to verify database initialization works as expected.
2. Run `data_cleaning.py` and ensure data is cleaned and updated correctly in the database.
